differ/differ_test.go

```diff
@@ -1056,7 +1056,7 @@ func TestProcessClustersNewIssuesNotPreviouslyNotified(t *testing.T) {
 
 	zerolog.SetGlobalLevel(zerolog.WarnLevel)
 	notifier = originalNotifier
-	previouslyReported = make(types.NotifiedRecordsPerCluster)
+	previouslyReported[types.NotificationBackendTarget] = make(types.NotifiedRecordsPerCluster)
```

Comment:
see comment in types/types.go.

Ideally there should be no need to map by eventTarget. Each instance should fetch the correct previouslyReported and work with that